qlib/WSDL.qm

```diff
@@ -4429,7 +4434,8 @@ public class WSDL::WebService inherits WSDL::XsdBase {
           - \c "binding": the binding of the port
      */
     list<hash<WSDL::ServiceInfo>> listServices() {
-        return cast<list<hash<WSDL::ServiceInfo>>>(services.values());
+        return map cast<hash<WSDL::ServiceInfo>>($1), services.values();
+        #return cast<list<hash<WSDL::ServiceInfo>>>(services.values());
```

Comment:
remove the commented code

Reply:
actually let me fix https://github.com/qorelanguage/qore/issues/2877 first and then do this right

Reply:
done